ibis/backends/tests/test_udf.py

```diff
@@ -6,6 +6,12 @@
 
 import ibis.common.exceptions as com
 from ibis import _, udf
+
+try:
+    from ibis.backends.pyspark import PYSPARK_LT_35
+except ImportError:
+    # Workaround when pyspark is not installed
+    PYSPARK_LT_35 = True
```

Comment:
```suggestion
```

Should be able to git rid of these if you use the `xfail_version` marker as suggested below.